py-polars/polars/functions/range/int_range.py

```diff
@@ -316,7 +317,25 @@ def int_ranges(
     │ 1     ┆ 3   ┆ [1, 2]     │
     │ -1    ┆ 2   ┆ [-1, 0, 1] │
     └───────┴─────┴────────────┘
+
+    `end` can be omitted for a shorter syntax.
+
+    >>> df = pl.DataFrame({"start": [3, -1], "end": [4, 2]})
+    >>> df.with_columns(int_range=pl.int_ranges("start"))
```

Comment:
This example doesn't make a lot of sense - why is there a start column? Better to use the existing `df` and select "end" and `int_range(end)`.

Reply:
I don't think using `end` is a good idea, it's confusing since I am saying just above that end can be omitted; what about this to demonstrate the short syntax?:

```
>>> df = pl.DataFrame({"l": [0, 1, 2, 3]})
    shape: (4, 1)
    ┌─────┐
    │ l   │
    │ --- │
    │ i64 │
    ╞═════╡
    │ 0   │
    │ 1   │
    │ 2   │
    │ 3   │
    └─────┘
    >>> df.select('l', pl.int_ranges('l'))
    shape: (4, 2)
    ┌─────┬───────────┐
    │ l   ┆ literal   │
    │ --- ┆ ---       │
    │ i64 ┆ list[i64] │
    ╞═════╪═══════════╡
    │ 0   ┆ []        │
    │ 1   ┆ [0]       │
    │ 2   ┆ [0, 1]    │
    │ 3   ┆ [0, 1, 2] │
    └─────┴───────────┘
```

Reply:
It's actually `start` being omitted. If you pass a single argument, `start` is set to `0`.

Reply:
Yes, I am talking about the `end=` parameter. Happy to rename `l` it to `end` if you prefer

Reply:
what about this:

```
    >>> df = pl.DataFrame({"end": [0, 1, 2, 3]})
    >>> df.with_columns(int_range=pl.int_ranges("end"))
    shape: (4, 2)
    ┌─────┬───────────┐
    │ end ┆ int_range │
    │ --- ┆ ---       │
    │ i64 ┆ list[i64] │
    ╞═════╪═══════════╡
    │ 0   ┆ []        │
    │ 1   ┆ [0]       │
    │ 2   ┆ [0, 1]    │
    │ 3   ┆ [0, 1, 2] │
    └─────┴───────────┘
```

Reply:
I went ahead and updated the example.